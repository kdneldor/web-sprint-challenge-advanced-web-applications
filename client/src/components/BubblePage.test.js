import React from "react";
import { render, screen } from "@testing-library/react";
import BubblePage from "./BubblePage";



test("Fetches data and renders the bubbles", () => {
  const { getByText } = render(<BubblePage />);
  const loading = getByText(/bubbles/i)
  expect(loading).toBeInTheDocument();
});







// const {queryAllByTestId, getAllByTestID, rerender} = render(
//   <Bubbles colors={[]} />
// )
// const initialColors = queryAllByTestId(/colors/i);
// expect(initialColors).toHaveLength(1);
// expect(initialColors).toEqual([]);

// rerender(<Bubbles error="" colors={data}/>)
// const colors = getAllByTestID(/colors/i)

// expect(colors).toHaveLength(10)


// const data = [
//   {
//     color: "aliceblue",
//     code: {
//       hex: "#f0f8ff"
//     },
//     id: 1
//   },
//   {
//     color: "limegreen",
//     code: {
//       hex: "#99ddbc"
//     },
//     id: 2
//   },
//   {
//     color: "aqua",
//     code: {
//       hex: "#00ffff"
//     },
//     id: 3
//   },
//   {
//     color: "aquamarine",
//     code: {
//       hex: "#7fffd4"
//     },
//     id: 4
//   },
//   {
//     color: "lilac",
//     code: {
//       hex: "#9a99dd"
//     },
//     id: 5
//   },
//   {
//     color: "softpink",
//     code: {
//       hex: "#dd99ba"
//     },
//     id: 6
//   },
//   {
//     color: "bisque",
//     code: {
//       hex: "#dd9a99"
//     },
//     id: 7
//   },
//   {
//     color: "softyellow",
//     code: {
//       hex: "#dcdd99"
//     },
//     id: 8
//   },
//   {
//     color: "blanchedalmond",
//     code: {
//       hex: "#ffebcd"
//     },
//     id: 9
//   },
//   {
//     color: "blue",
//     code: {
//       hex: "#6093ca"
//     },
//     id: 10
//   },
//   {
//     color: "blueviolet",
//     code: {
//       hex: "#8a2be2"
//     },
//     id: 11
//   }
// ];