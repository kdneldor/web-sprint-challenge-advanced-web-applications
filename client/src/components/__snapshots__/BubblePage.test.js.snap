// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Fetches data and renders the bubbles 1`] = `
<DocumentFragment>
  <div
    class="colors-wrap"
  >
    <p>
      colors
    </p>
    <ul />
    <div
      class="spacer"
    />
  </div>
  <div
    class="bubble-wrap"
  >
    <p>
      bubbles
    </p>
    <svg
      height="400"
      width="400"
    >
      <g />
    </svg>
  </div>
</DocumentFragment>
`;
